bot/src/main/java/com/intern/bot/service/TelegramBotService.java

```diff
@@ -0,0 +1,47 @@
+package com.intern.bot.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.telegram.telegrambots.meta.api.objects.Message;
+import reactor.core.publisher.Mono;
+
+@Service
+@RequiredArgsConstructor
+public class TelegramBotService {
+
+  private final WebClient webClient;
+  private final ObjectMapper objectMapper;
+  private final KafkaTemplate<String, Object> kafkaTemplate;
+
+  public Mono<String> getTopByMessageAmount(Long chatId) {
+    return webClient
+        .get()
+        .uri("/mongo/top/5?chatId={chatId}", chatId)
```

Comment:
Move it to properties. The reason behind it is that sometimes REST contract of the dependent service can be changed, so they bump up the version of the API like so `v2/mongo/top...` or add some new properties. So you should be ready for that and instead of changing code and pushing it into repo, you can add environment variable which overrides this value with newest endpoint.